stac_fastapi/pgstac/app.py

```diff
@@ -93,25 +93,31 @@
     "pagination": TokenPaginationExtension(),
 }
 
-known_extensions = {
-    *application_extensions_map.keys(),
+enabled_extensions = {
     *search_extensions_map.keys(),
     *cs_extensions_map.keys(),
     *itm_col_extensions_map.keys(),
     "collection_search",
 }
 
-enabled_extensions = (
-    os.environ["ENABLED_EXTENSIONS"].split(",")
-    if "ENABLED_EXTENSIONS" in os.environ
-    else known_extensions
-)
+if ext := os.environ.get("ENABLED_EXTENSIONS"):
+    enabled_extensions = set(ext.split(","))
 
-application_extensions = [
-    extension
-    for key, extension in application_extensions_map.items()
-    if key in enabled_extensions
-]
+application_extensions = []
+
+if "transaction" in enabled_extensions:
+    application_extensions.append(
+        TransactionExtension(
+            client=TransactionsClient(),
+            settings=settings,
+            response_class=ORJSONResponse,
+        ),
+    )
+
+if "bulk_transactions" in enabled_extensions:
```

Comment:
Different plurality than `transaction`?

```suggestion
if "bulk_transaction" in enabled_extensions:
```

Reply:
agreed, but is it worth changing if there are users out there that are setting `ENABLED_EXTENSIONS=bulk_transactions`?

Reply:
Since this a breaking change, maybe accept both but warn a [FutureWarning](https://docs.python.org/3/library/exceptions.html#FutureWarning) for the plural one, then remove sometime later?

Reply:
I guess this comes from https://github.com/stac-utils/stac-fastapi/blob/main/stac_fastapi/extensions/stac_fastapi/extensions/third_party/bulk_transactions.py while `transaction` is singular

Reply:
😓 both plural and singular are used in the classes names (`BaseBulkTransactions` `BulkTransactionExtension` )

Reply:
🫠 ok, maybe it's a can of worms not worth fixing now.